openedx/core/djangoapps/django_comment_common/comment_client/course.py

```diff
@@ -87,3 +87,26 @@ def get_course_user_stats(course_key: CourseKey, params: Optional[Dict] = None)
             "function:get_course_user_stats",
         ],
     )
+
+
+@function_trace("update_course_users_stats")
+def update_course_users_stats(course_key: CourseKey) -> Dict:
+    """
+    Update the user stats for all users for a particular course.
+
+    Args:
+        course_key (str|CourseKey): course key for which stats are needed.
+
+    Returns:
+        dict: data returned by API. Contains count of users updated.
+    """
+    url = f"{settings.PREFIX}/users/{course_key}/update_stats"
+    return perform_request(
+        'post',
+        url,
+        metric_action='user.update_course_stats',
+        metric_tags=[
+            f"course_key:{course_key}",
+            "function:update_course_users_stats",
+        ],
+    )
```

Comment:
@xitij2000 might be worth renaming `update_course_users_stats` to `update_course_discussion_users_stats` in the function here as well!

Reply:
This code is already namespaced to the discussions. The reason for renaming the management command is that when running `python manage.py lms update_course_user_stats` it's not clear that those stats are related to discussions. Here it's very clear since this code needs to be imported and used from the discussions module.